docker-compose.yml

```diff
@@ -32,15 +39,23 @@ services:
       - POSTGRES_HOST=postgres
       - POSTGRES_DATABASE=automatisch
       - POSTGRES_USERNAME=automatisch_user
+      - POSTGRES_PASSWORD=automatisch_password
+      - ENCRYPTION_KEY=automatisch_encryption_key
+      - APP_SECRET_KEY=automatisch_app_secret_key
+      - WORKER=true
     volumes:
       - automatisch_storage:/automatisch/storage
   postgres:
-    image: 'postgres:14.5'
+    image: 'postgres:14.5'    
     environment:
-      POSTGRES_HOST_AUTH_METHOD: trust
-      POSTGRES_DB: automatisch
-      POSTGRES_USER: automatisch_user
+      - POSTGRES_DB=automatisch
+      - POSTGRES_USER=automatisch_user
+      - POSTGRES_PASSWORD=automatisch_password
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
```

Comment:
When I run the compose def. as `docker compose up`, it cannot find `postgres_data` and `redis_data` volumes. I wonder if I am missing anything here.

However, it works when I add the missing volumes below in the `volumes` part of the compose definition.

Reply:
yes I was missing them. I used bindmounts in local testing so it wasn't needed.